Backend/models/User.ts

```diff
@@ -0,0 +1,27 @@
+import { sequelize } from "../lib/sequelize";
+import { DataTypes } from "sequelize";
+
+export const User = sequelize.define("User", {
+    id: {
+        type: DataTypes.INTEGER,
+        autoIncrement: true,
+        primaryKey: true,
+    },
+    firstName: {
+        type: DataTypes.STRING(50),
```

Comment:
there shouldn't be a limit on the name

Reply:
Should I use a TEXT type here?

Reply:
I've used STRING type which by default creates a field in Postgres with VARCHAR(255).